tests/Transports.Subscriptions.WebSockets.Tests/Transports.Subscriptions.WebSockets.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;netcoreapp2.2;net471</TargetFrameworks>
```

Comment:
Probably not such a big loss, remembering the names of the projects (AspNet**Core**).

Reply:
👍 

If we really wanted to support net471 then we have to spend extra effort to get ubuntu testing working so it ignores the net471 framework during tests.

Reply:
No, it won't work. Don't even try. I remember that working on .NET Framework requires a completely different implementation of websockets.